src/gen/conditions.rs

```diff
@@ -1325,52 +1333,20 @@ fn test_extra_arg(
     test(&conds, &spend);
 }
 
-#[cfg(test)]
-#[rstest]
-#[case(ASSERT_SECONDS_ABSOLUTE, ErrorCode::AssertSecondsAbsolute)]
-#[case(ASSERT_SECONDS_RELATIVE, ErrorCode::AssertSecondsRelative)]
-#[case(ASSERT_BEFORE_SECONDS_ABSOLUTE, ErrorCode::AssertBeforeSecondsAbsolute)]
-#[case(ASSERT_BEFORE_SECONDS_RELATIVE, ErrorCode::AssertBeforeSecondsRelative)]
-fn test_seconds_exceed_max(#[case] condition: ConditionOpcode, #[case] expected_error: ErrorCode) {
-    // ASSERT_SECONDS_RELATIVE
-    assert_eq!(
-        cond_test(&format!(
-            "((({{h1}} ({{h2}} (123 ((({} (0x010000000000000000 )))))",
-            condition as u8
-        ))
-        .unwrap_err()
-        .1,
-        expected_error
-    );
-}
-
-#[cfg(test)]
-#[rstest]
-#[case(ASSERT_HEIGHT_ABSOLUTE, ErrorCode::AssertHeightAbsolute)]
-#[case(ASSERT_HEIGHT_RELATIVE, ErrorCode::AssertHeightRelative)]
-#[case(ASSERT_BEFORE_HEIGHT_ABSOLUTE, ErrorCode::AssertBeforeHeightAbsolute)]
-#[case(ASSERT_BEFORE_HEIGHT_RELATIVE, ErrorCode::AssertBeforeHeightRelative)]
-fn test_height_exceed_max(#[case] condition: ConditionOpcode, #[case] expected_error: ErrorCode) {
```

Comment:
these two tests were replaced by the tests at the bottom of the file, and generalized. Not only do they cover the case of exceeding the upper limit, but also the lower limit